src/Core/Services/StateService.cs

```diff
@@ -581,6 +581,26 @@ public async Task SetDisableFaviconAsync(bool? value, string userId = null)
             await SetValueAsync(key, value, reconciledOptions);
         }
 
+        public async Task ApplyDisableFaviconGloballyAsync(bool? value)
+        {
+            // TODO remove this method (ApplyDisableFaviconGloballyAsync) to restore per-account icon support
+            await CheckStateAsync();
+            if (_state?.Accounts == null)
+            {
+                return;
+            }
+            var activeUserId = await GetActiveUserIdAsync();
+            foreach (var account in _state.Accounts)
+            {
+                var uid = account.Value?.Profile?.UserId;
+                // skip active user (theme already set)
+                if (uid != null && uid != activeUserId)
+                {
+                    await SetDisableFaviconAsync(value, uid);
+                }
+            }
+        }
```

Comment:
This is using almost the same code as `Task ApplyThemeGloballyAsync(string value)`. We could refactor a bit to share some more code, given that the only thing that changes is the call in the `if`; like a method that has a Task as a parameter that will be triggered in the i

Reply:
Took a slightly different approach to keep everything contained within `StateService`

Reply:
I really like the new approach 😄